src/main/scala/org/scalacheck/Prop.scala

```diff
@@ -300,6 +300,9 @@ object Prop {
   /** Create a property from a boolean value */
   def apply(b: Boolean): Prop = if(b) proved else falsified
 
+  /** Create a prop that evaluates the by-name argument on each application. */
+  def suspend(p: => Prop): Prop = apply(prms => p(prms))
```

Comment:
In `Gen`, we call this method `delay`: https://github.com/rickynils/scalacheck/blob/master/src/main/scala/org/scalacheck/Gen.scala#L455. Do you think it would be sensible to use the same name in `Prop`?